src/qt/walletmodel.cpp

```diff
@@ -514,6 +514,8 @@ bool WalletModel::bumpFee(uint256 hash, uint256& new_hash)
     questionString.append("</td><td>");
     questionString.append(BitcoinUnits::formatHtmlWithUnit(getOptionsModel()->getDisplayUnit(), new_fee));
     questionString.append("</td></tr></table>");
+    questionString.append("<br /><br />");
```

Comment:
nit: Just `<br>` is used in the most codebase in this repository.

Reply:
@hebasto  yeah by just adding that  before table row will fix it  instead of putting it in append.

@prayank23 below line should fix it

` questionString.append<br/>(tr("WARNING: New inputs and outputs can be added in replacement transaction if necessary which affects privacy"));`

Reply:
Done. Replaced `<br />` with `<br>`